packages/cli/src/commands/wallet/delete.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import { flags } from '@oclif/command'
+
+import Command from '../../OpenVaultCommand'
+
+export default class WalletDeleteCommand extends Command {
+  static description = 'Delete wallet'
+  static flags = {
+    ...Command.flags,
+    id: flags.string({
+      description: 'wallet local ID',
+    }),
+    type: flags.string({
+      description: 'wallet local type (hd or ledger)',
+    }),
+  }
+
+  async run() {
+    const client = this.client
+    if (client == null) {
+      return
+    }
+
```

Comment:
Could you add a confirmation prompt here please? Just in can people "accidentally" run this command.
The `vault:delete` command has this logic, should be pretty similar.